src/resources/Organizations/Organization.ts

```diff
@@ -102,8 +104,10 @@ export default class Organization extends Resource {
         return this.api.get<PrivilegeModel[]>(`${Organization.baseUrl}/${API.orgPlaceholder}/privileges/me`);
     }
 
-    listApiKeysPrivileges() {
-        return this.api.get<PrivilegeModel[]>(`${Organization.baseUrl}/${API.orgPlaceholder}/privileges/apikeys`);
+    listApiKeysPrivileges(options: listApiKeysPrivilegesOptions = {filter: PrivilegeFilterType.ALL}) {
+        return this.api.get<PrivilegeModel[]>(
+            this.buildPath(`${Organization.baseUrl}/${API.orgPlaceholder}/privileges/apikeys`, options),
```

Comment:
This endpoint does not seem to accept any query parameter according to [the swagger documentation](https://platform.cloud.coveo.com/docs/?urls.primaryName=AuthorizationServer#/Organization%20Privileges/rest_organizations_paramId_privileges_apikeys_get)